teams/nomis/database_pipeline.tf

```diff
@@ -123,8 +123,10 @@ resource "aws_imagebuilder_distribution_configuration" "database" {
       # }
 
       ami_tags = {
-        Name = local.database_pipeline.distribution.ami_name
+        Name          = "${local.database_pipeline.distribution.ami_name}${var.BRANCH_NAME == "main" ? "" : "-in-development"}"
```

Comment:
would it be better, if we added branch name to the AMI name, this way we can easily find out own AMIs, if there is a couple of us working on it;
although I'm sure it will be tricky with the AMI naming restrictions (e.g. no `/`)

Reply:
I considered that but branch names could be REALLY long. So the RHEL7 AMI built from this branch would be named something like nomis_RHEL7-9_BaseImage-nomis/add-branch-tfvar-for-ansible.

Reply:
yep, understand, maybe git username then?